early_access/master.adoc

```diff
@@ -19,8 +19,36 @@ For {productname} documentation, you should know that:
 * Prior to version 2.9.2, the product was referred to as Quay Enterprise
 endif::downstream[]
 
+include::modules/rn_3_80.adoc[leveloffset=+1]
+
+////
+[id="quay-config-fields-38"]
+== Configuration field updates 
+////
+
+include::modules/config-updates-38.adoc[New configuration fields]
+
+////
+[id='quay-release-notes']
+== {productname} 3.7 Release Notes
+
+{productname} is regularly released, containing new features, bug fixes, and software updates.
+We highly recommend deploying the latest version of {productname}.
```

Comment:
See "recommend" in SSG https://redhat-documentation.github.io/supplementary-style-guide/#_r

I am not sure if this is the case for quay, but you could say something like (you could probably come up with something better than me here:
```suggestion
Deploy the latest version of {productname} to ensure you are using the latest features, bug fixes, and software updates.
```

Reply:
Good catch. This was one of those "copy-overs".